docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory into the container (`.:/app`) can lead to potential issues, such as overwriting files in the container or including unnecessary files. Consider specifying only the necessary directories or files.